recipes/libev/all/patches/0001-cmake-for-msvc.patch

```diff
@@ -0,0 +1,254 @@
+From bb07532831ddb1fb379e2314d96b54e9dd601049 Mon Sep 17 00:00:00 2001
+From: Jihadist <tomasiche@gmail.com>
+Date: Wed, 9 Nov 2022 20:48:19 +0300
+Subject: [PATCH 2/2] cmake for msvc
+
+---
+ CMakeLists.txt       |  30 ++++++++++
+ config.h             | 128 +++++++++++++++++++++++++++++++++++++++++++
+ internal_utils.cmake |  65 ++++++++++++++++++++++
+ 3 files changed, 223 insertions(+)
+ create mode 100644 CMakeLists.txt
+ create mode 100644 config.h
+ create mode 100644 internal_utils.cmake
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+new file mode 100644
+index 0000000..9a94160
+--- /dev/null
++++ b/CMakeLists.txt
```

Comment:
For this case we typically do not use a patch, you can check this file in to this repo and then use https://github.com/conan-io/conan-center-index/blob/f53d2acfd5e3bc3bb27b7a3bfd118fc456405263/recipes/luau/all/conanfile.py#L49

to build with it 😄

Reply:
Is there any example? I'm not sure how to do it correctly

Reply:
Please follow the link, I put a recipe that does it :)